hack/chains/rekor-image-lookup.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash -ue
+
+DEFAULT_URL=quay.io/sbaird/chains-demo
+IMAGE_URL=${1:-$DEFAULT_URL}
+
+# First find the digest of the image
+IMAGE_DIGEST=$( skopeo inspect docker://$IMAGE_URL | jq -r .Digest )
+
+# Use the digest to do a rekor lookup
+UUIDS=$( rekor-cli search --sha "$IMAGE_DIGEST" 2>/dev/null )
```

Comment:
This lookup method was first described in [HACBS-75](https://issues.redhat.com/browse/HACBS-75) - thanks @caugello .